workloads/network-perf/run_serviceip_network_policy_test_fromgit.sh

```diff
@@ -9,15 +9,16 @@ if [[ "${isBareMetal}" == "true" ]]; then
   export METADATA_TARGETED=true
 fi
 
+export UUID=$(uuidgen)
 for pairs in 1 2 4; do
```

Comment:
I think we should add MOAR vars to the script. Missing a PAIRS vars :)

We can address that in a different PR though

Reply:
Yeah I noticed that as well. +1 to in another PR